src/github/getEvent.ts

```diff
@@ -13,7 +13,7 @@ declare global {
 
 export const getEvent = (request: HttpRequest): GithubEvent => {
   const rawBody = (request.rawBody as string) || '{}';
-  const signature: string = request.headers['x-hub-signature'];
+  const signature: string = request.headers['x-hub-signature-256'];
```

Comment:
just for your information, the new signature might not exist on current GitHub Enterprise Server versions. It might exist on the latest version, but I'm not 100% sure. To be save, you could do

```js
  const signature: string = request.headers['x-hub-signature-256'] || request.headers['x-hub-signature'];
```

Reply:
Good to know, cheers!